flink-cdc-runtime/src/main/java/org/apache/flink/cdc/runtime/operators/schema/distributed/SchemaCoordinator.java

```diff
@@ -131,6 +132,17 @@ public void start() throws Exception {
                 "Started SchemaRegistry for {}. Parallelism: {}", operatorName, currentParallelism);
     }
 
+    @Override
```

Comment:
Can we escalate Metadata Applier closing code to `SchemaRegistry#close`? Seems `metadataApplier` is also a field in abstract base class anyway.

Reply:
+1